README.md

```diff
@@ -1,2 +1,74 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+Submission of: Paul Rauca
+
+# Pseudocode
```

Comment:
The worksheet asked for just the pseudocode for the similarity calculation -- you have done more than was required here.